docs/uninstall-olm.md

```diff
@@ -0,0 +1,53 @@
+# How do I uninstall OLM?
+
+You can uninstall OLM in a symmetrical way as you did to install it. 
+
+## Uninstall Released OLM
+For uninstalling release versions of OLM, for example version 0.13.0, you can use the following command:
```

Comment:
SUPER nit: There should be a newline surrounding headers.

Reply:
smaller nit: To uninstall a specific release of OLM you can run the following commands:
```bash
export OLM_RELEASE=<olm-release-version>
...
```

Reply:
Another nit: `released versions`